alot/widgets/colour_text.py

```diff
@@ -0,0 +1,55 @@
+import logging
+import re
+from ..settings.const import settings
+
+
+def parse_text_colour(line):
+    """Return colour attribute as (attr, attr_focus)
+
+    :param str line: line of text to be parsed
+    :return: The theme attribute to apply
+    """
+    if settings.get('parse_quotes'):
+        return parse_quotes(line)
+
+
+def parse_quotes(line, max_quote_level=7):
```

Comment:
Maybe it is better to move the 7 into a local variable. The argument is not changed from anywhere (if I see that correctly). Please also document that there is the magic number 7 here because there are 7 levels predefined in the config spec. I think the code would work if a user defines a higher level in their config that is not in the spec. But I don't think we want to support this case.

Please fix the newline in the docstring. It should be between the text and the parameters not the params and the return.

Reply:
I pushed another commit to move the 7 to a local variable. I explain in the docstring up to what level we go and in a comment why is that.

From what I have tested, if a user defines `quote_level_8` (`>7`) in their theming config, that attribute is not considered. When a line is quoted, my code will match the highest possible quote level up to 7, then apply the theming attribute of that highest quote level found.

For instance:

    This is not a quoted line
    >>>>>>> This is 7th quote level
    >>>>>>>> This is 8th quote level
    >>>>>>>>> This is 9th quote level

Except for the not quoted line, they will all match the theming attribute of the 7th quote level. If no such attribute was defined in the theming config or if the line is not quoted, then the return value of `settings.get_theming_attribute` is `'default'`. If such a value is returned, we use the `theme.body` theming attribute instead.